languages/java/Makefile

```diff
@@ -16,7 +16,7 @@ test: rust
 	cd oso && mvn $(MVN_FLAGS) clean test
 
 package: rust
-	cd oso && mvn $(MVN_FLAGS) clean package
+	cd oso && mvn $(MVN_FLAGS) clean install
```

Comment:
Just so I could use the locally built `0.26.5-SNAPSHOT` in other projects like a fork of https://github.com/osohq/oso/issues/1655